docs/reference/commandline/service_create.md

```diff
@@ -734,6 +734,9 @@ The swarm extends my-network to each node running the service.
 Containers on the same network can access each other using
 [service discovery](https://docs.docker.com/engine/swarm/networking/#use-swarm-mode-service-discovery).
 
+Long form syntax of `--network` (since v17.06) allows to specify list of aliases and driver options:  
```

Comment:
Generally we don't mention specific versions (but we should back port this change to the 17.06 and up docs archive)